Aztec/Classes/Converters/ElementsToAttributedString/Implementations/VideoElementConverter.swift

```diff
@@ -52,10 +52,37 @@ class VideoElementConverter: AttachmentElementConverter {
             posterURL = nil
         }
 
-        let attachment = VideoAttachment(identifier: UUID().uuidString, srcURL: srcURL, posterURL: posterURL)
+        let sources = extractSources(from: element)
+
+        let attachment = VideoAttachment(identifier: UUID().uuidString, srcURL: srcURL, posterURL: posterURL, sources: sources)
 
         attachment.extraAttributes = extraAttributes
 
         return attachment
     }
+
+    func extractSources(from element: ElementNode) -> [VideoSource] {
```

Comment:
Should we perhaps just add a doc comment explaining what these methods do / why they're necessary?